src/app/js/formats/table/TableView.js

```diff
@@ -77,20 +106,55 @@ class TableView extends Component {
             );
         };
 
```

Comment:
rename id ot columnId
if(id !== this.state.sortId) return false;

return this.state.sort;